src/xdocs/config_annotation.xml

```diff
@@ -159,14 +166,14 @@ public String getNameIfPresent() { ... }
 &lt;/module&gt;
         </source>
         <p>
-          Example to allow one single parameterless annotation on the same line
+          Example to allow multiple annotations on the same line
         </p>
         <source>
-@Override public int hashCode() { ... } // no violations
-@SuppressWarnings("deprecation") public int foo() { ... } // violation
+@SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
         </source>
         <p>
-          Use the following configuration:
+          Use the following configuration to allow one single
+          parameterless annotation on the same line:
```

Comment:
Below recheck the config below. It looks the exact same as the default module. If it is the same, then we don't need 2 examples that show the same thing.

Reply:
Done

Reply:
Added a description saying that this configuration is same as default config.